dockerenforcer/docker_helper.py

```diff
@@ -163,3 +163,53 @@ def kill_container(self, container):
             logger.error("Communication error when stopping container {0}: {1}".format(container.cid, e))
         except Exception as e:
             logger.error("Unexpected error when stopping container {0}: {1}".format(container.cid, e))
+
+    def get_recent_detections_with_all_violated_rules(self, jurek, judge):
```

Comment:
@piontec It was placed to docker_helper because this method need access to the protected attribute __params_cache.

As a solution we can create a getter for __params_cache and move the method content directly in @app.route callback (or create separate function for it)